packages/gafl-webapp-service/src/pages/terms-and-conditions/terms-and-conditions.njk

```diff
@@ -56,7 +72,7 @@
                   items: [
                     {
                       value: "yes",
-                      text: mssgs.terms_conds_item_agree
```

Comment:
Now you've removed this, please can you check whether it can be deleted from the language resource files?